src/components/RefinementList/RefinementList.js

```diff
@@ -108,9 +120,20 @@ class RefinementList extends React.Component {
       cssClassList.push(`${this.props.cssClasses.depth}${this.props.depth}`);
     }
 
+    const limit = this.state.isShowMoreOpen ? this.props.limitMax : this.props.limitMin;
+    const showmoreBtn =
+      this.props.showMore ?
+        <Template
+          onClick={() => this.handleClickShowMore()}
+          templateKey={'showmore-' + (this.state.isShowMoreOpen ? 'active' : 'inactive')}
+          {...this.props.templateProps}
+        /> :
+        undefined;
```

Comment:
(tested and both seems to trigger the same behavior when rendered, no extra `<div>`)